packages/fiber/src/shared/Canvas.native.tsx

```diff
@@ -60,15 +58,15 @@ class ErrorBoundary extends React.Component<{ set: React.Dispatch<any> }, { erro
 }
 
 export const Canvas = /*#__PURE__*/ React.forwardRef<View, Props>(
```

Comment:
@joshuaellis, I'm wanting to make types platform-agnostic, but this has to extend props for a View or a div (same with `Props` itself.

Is this something that Typescript can infer? The implementation/API is otherwise 1-1.

Reply:
I'm not sure that's quite a tricky one. I'm trying to think of an instance where someone else might have needed to do this and we could check it out. I guess it could just be a generic with no extends?